features/step_definitions/request.rb

```diff
@@ -102,7 +102,9 @@ def build_given(api_version, operation)
     undo_builder = build_undo_for(operation_name, given_api_instance)
 
     # enable unstable operation
-    given_configuration.unstable_operations[operation_name.to_sym] = true
+    if given_configuration.unstable_operations.has_key?(operation_name.to_sym)
```

Comment:
isn't `key?` prefered over `has_key?` ?

Reply:
I believe they're just the same, one being an alias of the other, and it comes down to pure preference. I personally read `has_key?` better

Reply:
Yes, we'll just need to make sure `rubocop` or other code formatters don't mark it as deprecated (default behavior)